README.md

```diff
@@ -81,3 +78,89 @@ $> raito run
+| `sf-database-roles`                         | If set, database-roles for all databases will be fetched.                                                                                                                                                                                                                                                                                                                                                                                       | False     | `false`              |
+
+## Supported features
+
+| Feature             | Supported | Remarks                                |
+|---------------------|-----------|----------------------------------------|
+| Row level filtering | ✅         | Only supported for enterprise editions |
+| Column masking      | ✅         | Only supported for enterprise editions |
+| Locking             | ✅         | Support for both who and what lock     |
+| Replay              | ✅         | Explicit deletes cannot be replayed    |
+| Usage               | ✅         |                                        |
+
+## Supported dataobjects
+- Account (is represented as Raito DataSource)
+- Warehouse
+- Database
+- Shared Database
+- Schema
+- Shared Schema
+- Table
+- External Table
+- Shared Table
+- View
+- Materialized View
+- Shared View
+- Column
+- Shared Column
+
+
+## Access controls
+### From Target
+#### Account Roles
+Account roles are imported as `grant` with type `Role`.
+All grants to users and other roles associated with the role are added as who-items to the grant that will be imported in Raito.
+All grants to privileges on data objects associated the role are added as what-items to the grant that will be imported in Raito.
+All system roles (`ORGADMIN`, `ACCOUNTADMIN`, `SECURITYADMIN`, `USERADMIN`, `SYSADMIN`, `PUBLIC`) will be annotated as non-internalizable.
+If roles are managed by external identity stores in Snowflake, the following locks will be set: `name-lock`, `delete-lock`, `who-lock`, `inheritance-lock`.
+
+#### Database Roles
+Database roles are imported as `grant` with type `databaseRole`.
+All grants to users and other roles associated with the role are added as who-items to the grant that will be imported in Raito.
+All grants to privileges on data objects associated the role are added as what-items to the grant that will be imported in Raito.
+The who and what of the grants created by database roles will be locked and those grants.
+
+#### Masking Policies
+Masking policies are imported as `mask`.
+Those are non-internalizable as we are not able to indicate the `who` of all masking policies in all cases.
```

Comment:
Change to: Masking policies are imported as non-internalizable because most existing masking policies cannot be correctly interpreted Raito.